### ISSUE TITLE
Moudular dependency problems on Fedora 31

### ISSUE THREAD

**Issue by renich** [osbuild/osbuild-composer#207]
```
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]: Modular dependency problems:
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 1: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module eclipse:latest:3120191219031631:d36a98d4-0.x86_64
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 2: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module exa:latest:3120190813195051:22d7e2a5-0.x86_64
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 3: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module gimp:2.10:3120191106095052:f636be4b-0.x86_64
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 4: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module meson:latest:3120191009081836:dc56099c-0.x86_64
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 5: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module ninja:latest:3120190304180949:f636be4b-0.x86_64
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:  Problem 6: conflicting requests
Jan 24 12:17:07 introdesk.g02.org osbuild-composer[8662]:   - nothing provides module(platform:f31) needed by module nodejs:10:3120190816104510:f636be4b-0.x86_64
```

I am trying to build an image using the cockpit web front-end. I only get these messages and the image never builds.

**Comment by major** [comment]
This should be fixed by #240, right?

**Comment by teg** [comment]
This was a problem of lack of isolation, fixed by cdd1912e780d25b2a6113d3a240f30f80839d44c.
